nexus/src/external_api/http_entrypoints.rs

```diff
@@ -5967,6 +5975,125 @@ async fn current_user_ssh_key_delete(
     apictx.external_latencies.instrument_dropshot_handler(&rqctx, handler).await
 }
 
+/// List instrumentation probes.
+#[endpoint {
+    method = GET,
+    path = "/v1/probes",
+    tags = ["probes"],
+}]
+async fn probe_list(
+    rqctx: RequestContext<Arc<ServerContext>>,
+    query_params: Query<PaginatedByNameOrId<params::ProjectSelector>>,
+) -> Result<HttpResponseOk<ResultsPage<ProbeInfo>>, HttpError> {
+    let apictx = rqctx.context();
+    let handler = async {
+        let opctx = crate::context::op_context_for_external_api(&rqctx).await?;
+        opctx.authorize(authz::Action::ListChildren, &authz::FLEET).await?;
+
+        let nexus = &apictx.nexus;
+        let query = query_params.into_inner();
+        let pag_params = data_page_params_for(&rqctx, &query)?;
+        let scan_params = ScanByNameOrId::from_query(&query)?;
+        let paginated_by = name_or_id_pagination(&pag_params, scan_params)?;
+        let project_lookup =
+            nexus.project_lookup(&opctx, scan_params.selector.clone())?;
+
+        let probes =
+            nexus.probe_list(&opctx, &project_lookup, &paginated_by).await?;
+
+        Ok(HttpResponseOk(ScanByNameOrId::results_page(
+            &query,
+            probes,
+            &|_, p: &ProbeInfo| match paginated_by {
+                PaginatedBy::Id(_) => NameOrId::Id(p.id),
+                PaginatedBy::Name(_) => NameOrId::Name(p.name.clone().into()),
+            },
+        )?))
+    };
+    apictx.external_latencies.instrument_dropshot_handler(&rqctx, handler).await
+}
+
+/// View an instrumentation probe.
```

Comment:
```suggestion
/// View instrumentation probe
```

etc. for the others. No articles, no periods

Reply:
Done